### PR TITLE
Autoscroll after mounting in case `preserveScrollTop` is `false`

### PR DESCRIPTION
**Pull request by zalmoxisus**
Related to https://github.com/zalmoxisus/redux-devtools-extension/issues/38

### CODE REVIEW

File: src/LogMonitor.js [hunk 1]
```diff
@@ -122,17 +137,7 @@ export default class LogMonitor extends Component {
...
-    if (!node) {
-      return;
-    }
-    if (this.scrollDown) {
-      const { offsetHeight, scrollHeight } = node;
-      node.scrollTop = scrollHeight - offsetHeight;
-      this.scrollDown = false;
-    }
-  }
+  componentDidUpdate = this.scroll;
```
Comment (gaearon) [inline src/LogMonitor.js]: This looks a tiny bit surprising, let's turn this into a regular declaration instead?

Comment (zalmoxisus) [inline src/LogMonitor.js]: Sure :)
By regular declaration you mean `componentDidUpdate() { this.scroll(); }`?

Comment (zalmoxisus) [inline src/LogMonitor.js]: If so, agree, in my case we pass `prevProps` and `prevState` to `this.scroll`, which is  unnecessary.

Comment (gaearon) [inline src/LogMonitor.js]: First question: yes. 
Second question: not sure I understand. Can it just take no arguments?

Comment (zalmoxisus) [inline src/LogMonitor.js]: Yes, sorry, I wasn't clear enough. I meant that one disadvantage of `componentDidUpdate = this.scroll` (as I did before) is that it creates an impression that `this.scroll` is called as `this.scroll(prevProps, prevState)` as `componentDidUpdate` passes those arguments.

**Comment by gaearon** [comment]
Out in 1.0.4.

**Pull request merged**